test/transactions/transaction.js

```diff
@@ -192,6 +192,23 @@ describe('transaction.js', () => {
 		});
 	});
 
+	describe('#createTransaction with data', () => {
+		const createTransaction = transaction.createTransaction;
+		let trs = null;
+
+		it('should create transaction with data', () => {
+			trs = createTransaction('58191285901858109L', 1000, 'secret', '', 'data');
```

Comment:
Let's store this address, amount and secret in variables since we reuse them, and so it's obvious what they are from the variable names.